### PR TITLE
Upgrade indexer ( frozen esdts tokens) 

### PR DESCRIPTION
**Pull request by miiu96**
## Reasoning behind the pull request
- Changed the encoding for `tokenProperties` from `outport` package
- Updated indexer version
## Proposed changes
- Upgrade the version of the `mx-chain-es-indexer` module : 
  PR from indexer module: https://github.com/multiversx/mx-chain-es-indexer-go/pull/200

## Testing procedure
a. Create an ESDT token ( 'fungible' type). 
b. Send some ESDT tokens to a new address 
c. Freeze ESDT tokens of that address 
d. Check if in `accountsesdt` index field `frozen` is present 

Do the same steps for an NFT. 

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -16,7 +16,7 @@ require (
 	github.com/mitchellh/mapstructure v1.5.0
 	github.com/multiversx/mx-chain-core-go v1.1.30
 	github.com/multiversx/mx-chain-crypto-go v1.2.5
-	github.com/multiversx/mx-chain-es-indexer-go v1.3.7
+	github.com/multiversx/mx-chain-es-indexer-go v1.3.8-0.20230119093421-737bfdc6bf9d
```
Comment (bogdan-rosianu) [inline go.mod:19]: proper tag before merging

Comment (iulianpascalau) [reply go.mod:19]: proper release

Comment (miiu96) [reply go.mod:19]: will add

**Review by iulianpascalau — APPROVED** [review]

**Review by gabi-vuls — COMMENTED** [review]
[System-test-js-reports.zip](https://github.com/multiversx/mx-chain-go/files/10466500/System-test-js-reports.zip)

**Review by bogdan-rosianu — APPROVED** [review]

**Pull request merged**